pkg/models/active_node.go

```diff
@@ -2,10 +2,11 @@ package models
 
 //ActiveNode model
 type ActiveNode struct {
-	Id        int64  `json:"id"`
-	NodeId    string `json:"node_id"`
-	Heartbeat int64  `json:"heartbeat"`
-	Sequence  int32  `json:"sequence"`
+	Id           int64  `json:"id"`
+	NodeId       string `json:"node_id"`
+	Heartbeat    int64  `json:"heartbeat"`
+	Sequence     int32  `json:"sequence"`
```

Comment:
Suggestion: column name sequence sound little confusing, especially since there could be same number for different heartbeats. Should we change it to partition_id or something?